lib/utils.pm

```diff
@@ -6,7 +6,12 @@ use strict;
 use testapi;
 use File::Basename qw(basename);
 
-our @EXPORT = qw(switch_to_x11 wait_for_desktop ensure_unlocked_desktop wait_for_container_log);
+our @EXPORT = qw(clear_root_console switch_to_x11 wait_for_desktop ensure_unlocked_desktop wait_for_container_log);
+
+sub clear_root_console {
+    enter_cmd("clear");
+    assert_screen "root-console";
```

Comment:
please prefer single tickets as long as you don't want perl code to be evaluated. So better

```suggestion
    enter_cmd('clear');
    assert_screen 'root-console';
```

Reply:
seems you overlooked some other changes where there are still double ticks. Please check again (or I did something wrong?)